sdk/python/kfp/cli/run.py

```diff
@@ -140,3 +146,13 @@ def _print_runs(runs, output_format):
     headers = ['run id', 'name', 'status', 'created at']
     data = [[run.id, run.name, run.status, run.created_at.isoformat()] for run in runs]
     print_output(data, headers, output_format, table_format='grid')
+
+
+def _print_run_in_detail(client, run_id):
```

Comment:
Maybe this can be merged into `_print_runs`?

Reply:
maybe this function could be merged into `_display_run`
`_print_runs` and `print_run_in_detail` are not that similar I think
WDYT ?

Reply:
Maybe something like this:
(Do not support watch with detail)

```
def _display_run(client, namespace, run_id, watch, output_format, detail=False):
    run = client.get_run(run_id).run

    if detail:
        data = {
            key: value.isoformat() if isinstance(value, datetime.datetime) else value
            for key, value in run.to_dict().items()
            if key not in ['pipeline_spec']  # useless but too much detailed field
        }
        click.echo(data)
        return

    _print_runs([run], output_format)
    if not watch:
        return
    argo_path = shutil.which('argo')
...
...

```

Reply:
Could you re-check ? @chensun Thanks in advance!